packages/backend/src/ee/services/AiAgentService.ts

```diff
@@ -362,7 +366,11 @@ export class AiAgentService {
             ...metricQuery.metrics,
         ];
 
-        validateSelectedFieldsExistence(explore, metricQueryFields);
+        validateSelectedFieldsExistence(
+            explore,
+            metricQueryFields,
+            metricQuery.additionalMetrics,
```

Comment:
I was thinking of just passing the `metricQuery` param but this util is used in many places and doesn't justify the refactor.